JuliaExperimental/julia/zlattice.jl

```diff
@@ -9,13 +9,15 @@ module GAPZLattice
 
 import Core: Int, isa
 
-import Base: abs, convert, copy, deepcopy, diag, haskey, inv, lcm, length,
-             lexless, map, push!, sign, size, sum, trunc, zero, zeros
+import LinearAlgebra: diag
+
+import Base: abs, convert, copy, deepcopy, haskey, inv, lcm, length,
+             map, push!, sign, size, sum, trunc, zero, zeros
 
 export LLLReducedGramMat, ShortestVectors, OrthogonalEmbeddings
 
 
-doc"""
+raw"""
```

Comment:
raw?

Reply:
The problem is the `$` later in the string, which Julia tries to interpolate, like in a shell script. We could remove that; but @sebasguts suggested this alternative change, which disables variable interpolation for the string.

Reply:
We could probably also do `\$`, but I would try to investigate into this when we actually start to build julia documentation.